core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -397,6 +397,7 @@ public class DefaultFileSystemMaster extends CoreMaster
   /** Thread pool which asynchronously handles the completion of persist jobs. */
   private java.util.concurrent.ThreadPoolExecutor mPersistCheckerPool;
 
+  /** Manager for the Active UFS sync process. */
```

Comment:
The comment you add here is literally the name of the class, `ActiveSyncManager`, so I don't see a need to add this comment.

Reply:
Thanks for your comment, @ssz1997 .
I'll delete this line later.